src/search/AdvancedSearchContainer.tsx

```diff
@@ -120,6 +121,8 @@ const mapDispatchToProps = (dispatch: Dispatch<RootAction>) =>
       dispatchUpdateClauses: clauses => searchActions.updateClauses(clauses),
       dispatchUpdateQueryString: queryString =>
         searchActions.updateQueryString(queryString),
+      dispatchInitWithPreSelectedClauses: () =>
```

Comment:
To be inline with the rest of the `dispatchX` props this would become `dispatchAddPreSelectedClauses ` (or `dispatchSetPreSelectedClauses`)

Reply:
Done! Didn't notice the pattern!